#152798776  Static Page Management

Merdoth opened the pull request:
#### What does this PR do?

- Adds Static Page Management to the existing application

#### Description of Task to be completed?

- Adds static page feature to help the admin manage the content of static pages

#### How should this be manually tested?

- run ```git clone``` this repository
- navigate to the project directory
- run ```meteor npm install```
- run ```reaction``` from the command line
- sign in as an admin with the required crendentials
- click on static page icon on the dashboard to create or manage static pages

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
- story type: feature
- story id: 152798776
- story title: Static Page Management
 

#### Screenshots (if appropriate)
![screen shot 2018-01-11 at 2 05 09 pm 2](https://user-images.githubusercontent.com/28455813/34826870-8b1abc94-f6d8-11e7-8bdc-9dc2f860254d.png)

#### Question
N/A

Merdoth committed:
feature(static-page-management): - add static-page-management features
- add staticPageView.less
-add staticPage.less
- update main.less
- update package.json

[Finishes #154204656]

temilaj reviewed (changes requested):
Good job raising this PR. 
Here a few things to consider. 
-1 there's only 1 commit for all of these file changes. Is this really atomic?
- 2 we hace merge conflicts in this commit. How can we fix this

Merdoth committed:
feature(static-page-management): - fix merge conflict in package.json

Merdoth committed:
feature(static-page-management): Static page management

- create schema to accomommodate static page features
- add design style for static page components
-  update babel-runtime package to a newer version
- install tinymce wysiwyg editor
- register new components into application
- update database collections

[Delivers #154204656]

temilaj commented:
You have a failing CI build. Can you look that up?

Merdoth committed:
feature(static-page-management):  -fix failing test.

[Delivers #154204656]
